tests/posix/fs/src/test_fs_file.c

```diff
@@ -16,12 +16,13 @@ static int test_file_open(void)
 {
 	int res;
 
-	res = open(TEST_FILE, O_RDWR);
+	res = open(TEST_FILE, O_RDWR | O_CREAT);
```

Comment:
A mior nit, but can you please sort O_RDWR to the end - they will be consistent with a change above, and a usual code style (main op, then RD/WR flags).

Reply:
OK

Reply:
Done.